pkg/flowkit/config/account.go

```diff
@@ -30,6 +30,18 @@ type Account struct {
 	Name    string
 	Address flow.Address
 	Key     AccountKey
+
+	// FromFile is the configuration file containing this account.
+	//
+	// This field is only set if the external "fromFile"
+	// syntax is used. Otherwise this field is empty.
+	//
+	// Ref: https://docs.onflow.org/flow-cli/security/#private-account-configuration-file
+	FromFile string
+	// saveAdvancedFormat is to save an account in advanced format even if it has default values as defined
+	// in isDefaultKeyFormat() in flowkit/config/json/account.go
+	// defaults to false
+	SaveAdvancedFormat bool
```

Comment:
added this field so that account is saved as advanced format even if it has default account values (otherwise it will be saved as simple format), as per discussion to save account as advanced format in private json files.